src/components/authpage/LoginPage.jsx

```diff
@@ -0,0 +1,147 @@
+
+export default function LoginPage() {
+    const [email, setEmail] = useState("");
+    const [emailError, setEmailError] = useState(false);
+    const [passwordError, setPasswordError] = useState(false);
+    const [password, setPassword] = useState("");
+    const [showPassword, setShowPassword] = useState(false);
+
+    // useDocumentTitle("Login")
+
+    const { error, loginUser } = useAuth();
+
+    const useGuestCredentials = () => {
+        setEmail("guestuser@gmail.com");
+        setPassword("guestcredentials123");
+        setTimeout(
+            () => loginUser("guestuser@gmail.com", "guestcredentials123"),
+            100
+        );
+    };
+
+    const loginHandler = () => {
+        if (!email.match(emailMatchPattern)) {
+            setEmailError(true);
+            return;
+        } else {
+            setEmailError(false);
+        }
+
+        if (password.length < 8) {
+            setPasswordError(true);
+            return;
+        } else {
+            setPasswordError(false);
+        }
+
+        loginUser(email, password);
+    };
+
+    return (
+        <main className="grand-main">
+            <div className="flex justify-content-center align-items-center h-100">
+                <div className="auth-container ">
+                    <h2 className="text-align-center font-black">Login</h2>
+                    <div className="input-wrapper p-y-1 w-100">
+                        <label htmlFor="email" className="input-label fw-600">
+                            Email:
+                        </label>
+                        <input
+                            id="email"
+                            type="email"
+                            value={email}
+                            className={
+                                "input-field" +
+                                (emailError ? " input-color-error" : " ")
+                            }
+                            onChange={(e) => setEmail(e.target.value)}
+                        />
+                        {emailError && (
+                            <span className="input-message">
+                                Please enter valid email
+                            </span>
+                        )}
+                    </div>
+                    <div className="input-wrapper p-y-1 w-100">
+                        <label
+                            htmlFor="password"
+                            className="input-label fw-600"
+                        >
+                            Password:
+                        </label>
+                        <input
+                            id="password"
+                            type={showPassword ? "text" : "password"}
+                            value={password}
+                            className={
+                                "input-field" +
+                                (passwordError ? " input-color-error" : " ")
```

Comment:
suggestion you can use string interpolation

Reply:
Thanks for this suggestion